bot/cogs/birthday/birthday_commands.py

```diff
@@ -63,12 +95,45 @@ async def birthday_admin(self, interaction: Interaction):
 
     @birthday_admin.subcommand(name='config', description='Configure birthday settings')
     async def birthday_config(self, interaction: Interaction,
-                              channel: TextChannel = SlashOption(name='channel',
-                                                                 description='Channel to post birthday messages in.')):
-        success = birthday_helper.update_settings(interaction.guild_id, channel.id)
-        if not success:
-            return await interaction.send(f'An error occurred when updating birthday settings.', ephemeral=True)
+                                birthday_channel: TextChannel = SlashOption(name='birthday_channel',
```

Comment:
Added a separate config option for a baby month milestone channel (to put those in a different channel than birthday messages if desired)